src/modules/logger/logged_topics.h

```diff
@@ -110,7 +110,16 @@ class LoggedTopics
 	 * @return true on success
 	 */
 	bool add_topic(const char *name, uint16_t interval_ms = 0, uint8_t instance = 0);
-	bool add_topic_multi(const char *name, uint16_t interval_ms = 0);
+
+	/**
+	 * Add a topic to be logged.
+	 * @param name topic name
+	 * @param interval limit in milliseconds if >0, otherwise log as fast as the topic is updated.
+	 * @param instance orb topic instance
+	 * @param max_instance the max multi-instance to add.
```

Comment:
This is fairly nitpicky, but I would understand this as the maximum instance index to add (`instance <= max_instance`), not the max count. `max_num_instances` would IMO be clearer.

Reply:
Please nitpick away, I wasn't sure about the naming here.

Reply:
Renamed.